app/src/main/java/com/keylesspalace/tusky/adapter/StatusViewHolder.java

```diff
@@ -91,20 +96,22 @@ public void setupWithStatus(@NonNull StatusViewData.Concrete status,
         super.setupWithStatus(status, listener, statusDisplayOptions, payloads);
     }
 
-    private void setRebloggedByDisplayName(final CharSequence name,
-                                           final List<Emoji> accountEmoji,
-                                           final StatusDisplayOptions statusDisplayOptions) {
+    private void setStatusInfoContent(final CharSequence name,
+                                      final boolean isReply,
+                                      final List<Emoji> nameEmoji,
+                                      final StatusDisplayOptions statusDisplayOptions) {
         Context context = statusInfo.getContext();
         CharSequence wrappedName = StringUtils.unicodeWrap(name);
-        CharSequence boostedText = context.getString(R.string.post_boosted_format, wrappedName);
+        CharSequence translatedText = context.getString(isReply ? R.string.post_replied_format : R.string.post_boosted_format, wrappedName);
         CharSequence emojifiedText = CustomEmojiHelper.emojify(
-                boostedText, accountEmoji, statusInfo, statusDisplayOptions.animateEmojis()
+                translatedText, nameEmoji, statusInfo, statusDisplayOptions.animateEmojis()
         );
         statusInfo.setText(emojifiedText);
+        statusInfo.setCompoundDrawablesWithIntrinsicBounds(isReply ? R.drawable.ic_reply_all_18dp : R.drawable.ic_reblog_18dp, 0, 0, 0);
```

Comment:
The reply_all icon needs to be tinted in the same color as the text. It is currently invisible in the white theme. The easiest way to do so is probably to change the `fillColor` to the font color attribute.

Reply:
Ok.